src/Controllers/SchemaController.js

```diff
@@ -648,6 +655,21 @@ const _IdempotencySchema = convertSchemaToAdapterSchema(
     classLevelPermissions: {},
   })
 );
+const _FileSchema = convertSchemaToAdapterSchema(
+  injectDefaultSchema({
+    className: '_File',
+    fields: defaultColumns._File,
+    classLevelPermissions: {
+      find: {
+        '*': true,
+      },
```

Comment:
I think here we should disable the find permission for security reasons because not every developers will take the time to ensure that the parent object acls are transfered on the file ACL.
Current configuration will be less secure than the current one if no ACL configured. May be we should add a default behavior: Parse.File automatically add ACL from parent objet to itself.

This way Parse.File will be only reachable based on the parent object.

Reply:
Good call!